Book-A-Desk.Domain/CommandHandler.Reservations.fs

```diff
@@ -26,18 +25,17 @@ module BookADeskCommandHandler =
 
             let run executeCommandWith cmd (ReservationId aggregateId) =
                 eventStore.GetEvents aggregateId
-                |> List.map (function | ReservationEvent event -> event | _ -> failwithf "There is an unexpected event type for AggregateId:%s" (aggregateId.ToString()))
```

Comment:
Note for reviewers: I removed this since they were generating warnings. I talked with Stefan and we thought it would be a good idea to try enabling Warning As Errors for the future to keep a clean code base since it's so young.

Reply:
Excellent idea, we add this to all of our projects: https://github.com/ayudasystems/org-manager-api/blob/master/Directory.Build.props